runtime/src/alpha/mod.rs

```diff
@@ -4,48 +4,44 @@ use std::{
     net::{Ipv4Addr, SocketAddr},
     ops::DerefMut,
     str::FromStr,
-    sync::{Arc, Mutex},
+    sync::Mutex,
     time::Duration,
 };
 
 use anyhow::Context;
 use async_trait::async_trait;
 use core::future::Future;
-use shuttle_common::{
-    backends::{
-        auth::{AuthPublicKey, JwtAuthenticationLayer},
-        trace::ExtractPropagationLayer,
-    },
-    claims::Claim,
-    resource,
-    secrets::Secret,
+use shuttle_common::{backends::trace::ExtractPropagationLayer, secrets::Secret};
+use shuttle_proto::runtime::{
+    runtime_server::{Runtime, RuntimeServer},
+    LoadRequest, LoadResponse, StartRequest, StartResponse, StopReason, StopRequest, StopResponse,
+    SubscribeStopRequest, SubscribeStopResponse,
 };
-use shuttle_proto::{
-    provisioner,
-    runtime::{
-        runtime_server::{Runtime, RuntimeServer},
-        LoadRequest, LoadResponse, StartRequest, StartResponse, StopReason, StopRequest,
-        StopResponse, SubscribeStopRequest, SubscribeStopResponse,
-    },
-};
-use shuttle_service::{Environment, Factory, Service};
+use shuttle_service::{Factory, Service};
 use tokio::sync::{
     broadcast::{self, Sender},
     mpsc, oneshot,
 };
 use tokio_stream::wrappers::ReceiverStream;
-use tonic::{
-    transport::{Endpoint, Server},
-    Request, Response, Status,
-};
+use tonic::{transport::Server, Request, Response, Status};
 
-use crate::__internals::{print_version, ProvisionerFactory, ResourceTracker};
+use crate::__internals::{print_version, ProvisionerFactory};
 
-use self::args::Args;
+use crate::args::args;
 
-mod args;
+// uses custom macro instead of clap to reduce dependency weight
+args! {
+    pub struct Args {
+        // The port to open the gRPC control layer on.
+        // The address to expose for the service is given in the StartRequest.
```

Comment:
This comment is no longer correct right?

Reply:
it is, the deployer just has the start input hardcoded to `0.0.0.0:8000`